### PR TITLE
Fraud mark only required on updates

### PR DESCRIPTION
**Pull request by theosato**
@lbiselli

### CODE REVIEW

File: src/Inspetor/Model/InspetorSale.php [hunk 1]
```diff
@@ -141,6 +137,10 @@ public function isValidUpdate() {
 			$this->validateStatus();
 		}
 
+		if (!$this->is_fraud) {
```
Comment (pearsonhenri) [inline src/Inspetor/Model/InspetorSale.php]: mmm what happens here if `is_fraud == False`?

File: tests/Inspetor/Model/InspetorSaleTest.php [hunk 2]
```diff
@@ -90,15 +90,15 @@ public function testIfIsNotValidWhenStatusIsNotValidIsNull() {
         $sale->isValid();
     }
 
-    public function testIfIsNotValidWhenIsFraudIsNull() {
-        $sale = $this->getDefaultSale();
-        $sale->setIsFraud(null);
+    // public function testIfIsNotValidWhenIsFraudIsNull() {
```
Comment (pearsonhenri) [inline tests/Inspetor/Model/InspetorSaleTest.php]: 🤔

Comment (lbiselli) [reply tests/Inspetor/Model/InspetorSaleTest.php]: We should change this to test the isValidUpdate

File: src/Inspetor/Model/InspetorSale.php (hunk 1 above)
Comment (lbiselli) [reply src/Inspetor/Model/InspetorSale.php]: Yeah! We should do `if (this->is_fraud == null)`

Comment (theosato) [reply src/Inspetor/Model/InspetorSale.php]: That's right. I saw that and "fixed it" in the python lib creation, but didn't fix here. NOOOOOICE.

File: tests/Inspetor/Model/InspetorSaleTest.php (hunk 2 above)
Comment (theosato) [reply tests/Inspetor/Model/InspetorSaleTest.php]: Someone (theo) forgot that.

**Pull request merged**